docs/roadmap-2021.md

```diff
@@ -2,6 +2,13 @@
 
 ## Next 3 Months (Short-term)
 
+### Knative Eventing 1.0
+Releasing Knative Eventing v1.0.
+
+**GitHub Issue** https://github.com/knative/eventing/issues/5039
+
+**Owner:** Scott Nichols
```

Comment:
Likely this should be an Eventing WG lead?

Reply:
I can take this.

Reply:
Done